README.md

```diff
@@ -125,7 +139,20 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S1        |    lazy   |     S1
+    S1        |    smelly |     S1
+    S1        |           |     S2
```

Comment:
This line says "IF we get nothing, transition from S1 to S2" where nothing doesn't mean EOI.

# -1